test/parallel/test-datetime-change-notify.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+const {
+  notifyDateTimeConfigurationChange,
+} = require('v8');
+
+process.env.TZ = 'UTC';
+notifyDateTimeConfigurationChange();
+assert.match(new Date().toString(), /GMT\+0000/);
```

Comment:
We may need to set multiple different values in the test to confidently ensure the notification does work (like say the value doesn't change).